R/IDateTime.R

```diff
@@ -7,6 +7,10 @@ as.IDate = function(x, ...) UseMethod("as.IDate")
 
 as.IDate.default = function(x, ..., tz = attr(x, "tzone", exact=TRUE)) {
   if (is.null(tz)) tz = "UTC"
+  # 4676 mimics for back-compatibility a similar patch applied to as.Date.character in r79119
```

Comment:
since changes are needed anyway, this comment seems more appropriate inside if the if().  Since the if() has been elongated to 3 lines anyway, may as well provide an extra reason for the elongation by putting the comment inside.